Changes

```diff
@@ -94,6 +94,12 @@ Working version
    review by Florian Angeletti, Anil Madhavapeddy, Gabriel Scherer,
    and Miod Vallat)
 
+- #?????: Document support for Linux perf and frame pointers.
```

Comment:
I believe `perf` should be in monospace, if it's the Linux command

Reply:
I wouldn't worry about this file unless I've made a spelling mistake :-)